docker/nginx.conf

```diff
@@ -128,15 +129,15 @@ http {
   }
 
   server {
-    listen [::]:8080;
-    listen 8080;
+    listen [::]:9999;
```

Comment:
this will break the image that gets built, as the production deployment is looking at 8080. making this configurable with an env var is probably the fastest method of resolving this, but you might want to look at the kind config and see if it's already configurable over there

Reply:
I'm super confused. This file was copy pasta'd in. IIRC, the original wasn't being used for anything so I think the previous port of 8080 was a vestigial port. I don't think you have a point about the env var resolving this though